spec/models/content_item_spec.rb

```diff
@@ -1,124 +1,75 @@
 require 'rails_helper'
 
 RSpec.describe ContentItem do
-  let(:content_item) do
-    ContentItem.new(
-      content_item_params
-    ).tap do |content_item|
-      content_item.link_set = Tagging::ContentItemExpandedLinks.new(link_set_params)
-    end
-  end
+  subject(:content_item) { described_class.new(data, blacklist: blacklist) }
```

Comment:
Using `subject` allows us to make more concise assertions such as:

```rb
it { is_expected.to include(:ordered_related_items_overrides) }
```

instead of:

```rb
it "should include :ordered_related_items_overrides" do
  expect(content_item.blacklisted_tag_types).to include(:ordered_related_items_overrides)
end 
```

IMHO it is now easier to see what changes between each test context and the effect that has on the method under test.